.github/workflows/jira-creation.yml

```
@@ -104,14 +104,14 @@ jobs:
         id: echo_inputs
         run: |
           echo "all variables defined as ouputs in previous jobs"
-          echo "Running for label:   ${{ matrix.label }}""
+          echo "Running for label:   ${{ matrix.label }}"
           echo "Issue ID:            ${{ needs.create-issue.outputs.issueId }}"
 
       - name: Setup Jira
         uses: atlassian/gajira-cli@v2.0.2
 
       - name: Login
-        uses: atlassian/gajira-login@v2.0.0
+        uses: atlassian/gajira-login@v2.0.1
```

```suggestion
        uses: atlassian/gajira-login@v2.0.2
```
Line 46 changes the version to 2.0.2 so making this the same for consistency

---

Or is the correct version 2.0.1 because I see you use that in 2 places now

---

Good catch.  That was meant for the `gajira-create` version.  Fixed `gajira-login` to point to 2.0.0. everywhere.